kura/org.eclipse.kura.rest.identity.provider/src/main/java/org/eclipse/kura/internal/rest/identity/provider/validator/PasswordStrengthValidators.java

```diff
@@ -0,0 +1,117 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2021 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.internal.rest.identity.provider.validator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PasswordStrengthValidators {
+
```

Comment:
I suppose that these validators come from the web2 bundle, right? We could move them in a separate bundle and import them. But probably it is out of scope for this PR...

Reply:
yes I had to import the validator classes to avoid a dependency with web that one day we will remove. Don't know if it makes sense create a new bundle used by web2 and this one since we are going to remove web2.